web/blog/the-new-rest-api/index.md

```diff
@@ -157,16 +170,191 @@ vast web server --mode=mtls
 ## Usage Examples
 
 Now that you know how we put the REST API together, let's look at some
-end-to-end examples. We built the REST API for two reasons::
+end-to-end examples.
+
+### See what's inside VAST
+
+One straightforward example is checking the number of records in VAST:
+
+```bash
+curl "https://vast.example.org:42001/api/v0/status?verbosity=detailed" \
+  | jq .index.statistics
+```
+
+```json
+{
+  "events": {
+    "total": 8462
+  },
+  "layouts": {
+    "zeek.conn": {
+      "count": 8462,
+      "percentage": 100
+    }
+  }
+}
+```
+
+### Perform a HTTP health check
+
+The `/status` endpoint can also be used as a HTTP health check in
+`docker-compose`:
+
+```yaml
+version: '3.4'
+services:
+  web:
+    image: tenzir/vast
+    environment:
+      - "VAST_START__COMMANDS=['web server --mode=dev']"
```

Comment:
This syntax is going away with VAST v3.0, where it's just comma-separated. As that already worked with VAST v2.4, I would recommend using that here.

```suggestion
      - "VAST_START__COMMANDS=web server --mode=dev"
```